test/unit-tests/testConviva.js

```diff
@@ -864,4 +864,118 @@ describe('Analytics Framework Conviva Plugin Unit Tests', function() {
     simulator.simulateVideoElementCreated("http://testStreamUrl");
     expect(Conviva.currentContentMetadata.applicationName).toBe(undefined);
   });
-});
\ No newline at end of file
+
+  it('Conviva Plugin should report ad errors', function()
+  {
+    var plugin = createPlugin(framework, {
+      "gatewayUrl":"testUrl",
+      "customerKey":"testKey",
+      "applicationName": {}
+    });
+    var simulator = Utils.createPlaybackSimulator(plugin);
+    var embedCode = "testEmbedCode";
+    var title = "testTitle";
+    simulator.simulatePlayerLoad({
+      embedCode: embedCode,
+      title: title,
+      duration: 60000
+    });
+    simulator.simulateVideoElementCreated("http://testStreamUrl");
```

Comment:
Can you pull and merge from master? There's a new function called startPlayer to simulate the video element creation and a new requirement to simulate the INITIAL_PLAYBACK_REQUESTED message.